data/blog/advancements-in-lightning-infrastructure.mdx

```diff
@@ -0,0 +1,372 @@
+across multiple implementations, successfully testing it between Core Lightning
+(CLN) and Eclair. By resolving protocol-level inconsistencies, his work marks a
+major step toward universal Splicing adoption across the Lightning ecosystem.
+
+Beyond direct interoperability, Dusty has introduced [Splice
+Script](https://github.com/ElementsProject/lightning/pull/6980), a structured
+scripting language that allows users to automate and optimize liquidity
+management tasks—such as batching on-chain movements or scheduling
+transactions based on fee conditions. The integration of its parser, compiler,
+and validator into CLN has transformed Splice Script into a practical tool for
+both node operators and application developers. Additionally, the Splice Solver
+component helps streamline transaction construction, reducing errors, and making
+complex Splicing operations more accessible.
+
+The impact of these enhancements goes beyond just payment channels. By making
+on-chain transaction logic more efficient, Splicing and Splice Script pave the
+way for improved privacy in Bitcoin transactions, including
+[Coinjoins](https://bitcoinops.org/en/topics/coinjoin/) and
+[Payjoins](/blog/developing-advancements-in-onchain-privacy#async-payjoin). As
+Dusty explains:
+
+>The dream of higher-privacy decentralized on-chain coin sharing is brought
+>ever so closer to reality with these improvements. These privacy projects have
+>always been separate from Lightning but Splicing enables them to be merged
+>together, creating a larger base from which to derive anonymity.
+
+As regulatory pressures mount, building resilient, decentralized infrastructure
+is more important than ever. These enhancements reinforce self-custody and
+censorship resistance to ensure Bitcoin remains a tool for financial freedom.
+
+In addition to supporting Dusty's work on Splicing, OpenSats has also funded an
+initiative to integrate [Splicing into
+LDK](/blog/bitcoin-grants-feb-2024#splicing-in-ldk). Over the past several
+months, efforts to integrate Splicing into LDK have yielded meaningful strides
+that led to a key breakthrough: the integration of [Interactive Transaction
+Construction](https://github.com/lightningdevkit/rust-lightning/pull/2419). This
+feature closely ties in with [Dual
+Funding](https://github.com/lightningdevkit/rust-lightning/issues/1621#issue-1308076236),
+a mechanism that allows two participants to collaboratively open channels.
+Because Splicing and Dual Funding share underlying transaction logic, progress
+on one directly benefits the other.
+
+This work has required frequent releases to keep pace with LDK's rapid evolution
+and ensure compatibility with the latest protocol updates. In parallel, Splicing
+messages (including `splice_locked`) have been implemented, along with support
+for `Pay-After-Splice` and `Splice-After-Pay`—features that allow channels to
+be resized dynamically without interrupting active transactions.
```

Comment:
This paragraph should be dropped completely, as it contains minute details that don't add much info in this context.